packages/contracts/contracts/deploy/PayableDAppFactory.sol

```diff
@@ -50,6 +56,16 @@ contract PayableDAppFactory is
         transferOwnership(_owner);
     }
 
+    /// @notice sets a name for the pool using ENS service
```

Comment:
This comment can be removed, since it will inherit from the interface, if you ever use some tool to scrape the documentation off these contracts. Also, why "pool"?